src/ergw_aaa_radius.erl

```diff
@@ -244,6 +244,16 @@ validate_option(vendor_dicts, Value) when is_list(Value) ->
     lists:map(fun validate_vendor/1, Value);
 validate_option(termination_cause_mapping, Value) ->
     validate_termination_cause_mapping(Value);
+validate_option(server, #{default := DefVal, backup := BkupVal} = Dests) ->
```

Comment:
With only two keys it is possible to select them like this, but consider using maps:fold/3 instead.
Also, is it correct to have only backup? Without default. Sounds strange.

Reply:
Agree, when I pickup this PR the main point was rebase it with master, but logic looks like really a strange, will be fixed